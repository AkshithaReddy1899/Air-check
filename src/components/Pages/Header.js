import React from 'react';

const Header = () => (
  <nav className="main-nav">
    <div>
      <i className="fas fa-chevron-left" />
      {new Date().getFullYear()}
    </div>
    <div>
      Countries Details
    </div>
    <div>
      <i className="fas fa-microphone" />
      &nbsp;&nbsp;&nbsp;
      <i className="fas fa-cog" />
    </div>
  </nav>
);

export default Header;
