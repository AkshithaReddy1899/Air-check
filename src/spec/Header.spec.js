import React from 'react';
import { Provider } from 'react-redux';
import matches from './test-utils';
import store from '../redux/configStore';
import Header from '../components/Pages/Header';

describe('Header', () => {
  it('Header matched snapshot', () => {
    matches(<Provider store={store}><Header /></Provider>);
  });
});
