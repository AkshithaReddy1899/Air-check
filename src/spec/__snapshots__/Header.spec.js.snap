// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Header matched snapshot 1`] = `
<nav
  className="main-nav"
>
  <div>
    <i
      className="fas fa-chevron-left"
    />
    2022
  </div>
  <div>
    Countries Details
  </div>
  <div>
    <i
      className="fas fa-microphone"
    />
       
    <i
      className="fas fa-cog"
    />
  </div>
</nav>
`;

exports[`HomePage Homepage matched snapshot 1`] = `
<nav
  className="main-nav"
>
  <div>
    <i
      className="fas fa-chevron-left"
    />
    2022
  </div>
  <div>
    Countries Details
  </div>
  <div>
    <i
      className="fas fa-microphone"
    />
       
    <i
      className="fas fa-cog"
    />
  </div>
</nav>
`;
